db_stress_tool/db_stress_gflags.cc

```diff
@@ -674,4 +674,8 @@ DEFINE_int32(approximate_size_one_in, 64,
 
 DEFINE_int32(read_fault_one_in, 1000,
             "On non-zero, enables fault injection on read");
+
+DEFINE_bool(env_fault_injection, false,
```

Comment:
Can we leave env out of the name? Maybe name it ```write_fault_injection``` or ```sync_fault_injection```. The reason is we're moving to the FileSystem class for file related APIs, and this might become a bit of a misnomer.

Reply:
Sure.  Let me update to apply only FaultInjectionFS.